### DISCUSSION TITLE
Optimum package for nextjs server to planetscale db

### DISCUSSION THREAD

**Discussion by MatthewPosgate** [planetscale/discussion#609] [General]
My experience with databases over the years has lead me to want to reduce the number of round trips to the database as much as possible.
I want to be able to submit semi-colon separated sql to the database in one go.
I'm currently using Nextjs, not on the edge.  All the docs point me at @planetscale/database but from what I can see this only allows me to submit one statement at a time. Are there other options?

**Comment by mscoutermarsh** [comment]
> not on the edge

This gives you many more options. You can use any library that is mysql compatible. I'm not familiar enough with all the libraries to know which is best, but hoping that points you in the right direction.

**Reply by MatthewPosgate** [reply]
Thanks. I'm using the mysql2 package and it's working fine
